bravado/swagger_model.py

```diff
@@ -6,8 +6,8 @@
 import yaml
 
 from bravado_core.spec import is_yaml
-from six.moves import urllib
-from six.moves.urllib import parse as urlparse
+import urllib.request, urllib.parse, urllib.error
```

Comment:
Why not 'from urllib import request, parse, error' ??

Reply:
because they have another method called request and I wanted to do minimum changes possible as this is forked repo

Reply:
done